VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -1378,13 +1381,27 @@ Function Install-EsxiCertificate {
             if (!$esxiHosts) { Write-Error "No ESXi host $esxiFqdn found in workload domain $domain." -ErrorAction Stop }
         }
 
-        # Perform ESXi host vSAN data migration pre-check.
-        if ($PsBoundParameters.ContainsKey("cluster")) {
-            Write-Output "Performing Data Migration Pre-check on the cluster $cluster"
-            Get-EsxiHostVsanMaintenanceModePrecheck -server $server -user $user -pass $pass -domain $domain -cluster $cluster -vsanDataMigrationMode $vsanDataMigrationMode
+        $version = Get-VCFManager -version
+        $vcfVersion = $version.Split('.')[0]+"."+$version.Split('.')[1]
+
```

Comment:
-ge ?

Reply:
addressed in commit e6a7844965249faebca841b4da521bef3b7b40ba